<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenDialog</class>
 <widget class="QDialog" name="PenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStyle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Стиль:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выбрать цвет</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Толщина:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxWeight">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>70</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Заголовок</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStyle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Без линии</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Обычный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пунктир</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пунктир-точка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пунктир-точка-точка</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxStyle</tabstop>
  <tabstop>spinBoxWeight</tabstop>
  <tabstop>pushButtonColor</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PenDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PenDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
